.idea/runConfigurations.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please revert the deletion of this file?

Reply:
done with the next commit